### PR TITLE
Issue#529 welcome page

### PR DESCRIPTION
**Pull request by Hugo-Hontans**
Create the welcome page

### CODE REVIEW

File: libs/auth/src/lib/pages/welcome-view/welcome-view.component.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    fxLayoutAlign.lt-md="start center"
+    fxLayoutAlign="start start"
+  >
+    <h1>Welcome to Blockframes</h1>
+    <p>
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec luctus dictum metus quis
+      sagittis. Curabitur scelerisque neque erat, ac bibendum nisi aliquet et. Duis a auctor nulla,
+      a finibus mauris. Nullam eu maximus lorem.
+    </p>
+    <a routerLink="/auth/connexion" routerLinkActive="router-link-active">
```
Comment (GrandSchtroumpf) [inline libs/auth/src/lib/pages/welcome-view/welcome-view.component.html]: Where are you using : router-link-active class ?

File: libs/auth/src/lib/pages/welcome-view/welcome-view.component.scss [hunk 2]
```diff
@@ -0,0 +1,56 @@
+section {
+  height: 100%;
+  background-color: var(--whiteBackground);
+}
+
+h1 {
+  width: 308px;
```
Comment (GrandSchtroumpf) [inline libs/auth/src/lib/pages/welcome-view/welcome-view.component.scss]: Why 308px?

File: libs/auth/src/lib/pages/welcome-view/welcome-view.component.html (hunk 1 above)
Comment (Hugo-Hontans) [reply libs/auth/src/lib/pages/welcome-view/welcome-view.component.html]: It was my auto-complete....

File: libs/auth/src/lib/pages/welcome-view/welcome-view.component.scss (hunk 2 above)
Comment (Hugo-Hontans) [reply libs/auth/src/lib/pages/welcome-view/welcome-view.component.scss]: to wrap the title to the line, maybe it's not a good practice

Comment (GrandSchtroumpf) [reply libs/auth/src/lib/pages/welcome-view/welcome-view.component.scss]: Yes, try to keep things simple and not "pixel perfect" for normal CSS. If it's inside the UI Kit, then you can have very specific values. But for that, if you do that it'll break on responsive.

**Pull request merged**